config/schema/og.schema.yml

```diff
@@ -110,6 +110,9 @@ og.og_role.*:
     group_bundle:
       type: string
       label: 'Group bundle'
+    is_admin:
```

Comment:
Thanks @amitaibu ! And yes, that is pretty much how I interpreted the use of roles and permissions. Which is useful :)